application/src/main/java/net/corda/v5/application/messaging/FlowMessaging.java

```diff
@@ -106,6 +106,25 @@ public interface FlowMessaging {
     @NotNull
     FlowSession initiateFlow(@NotNull MemberX500Name x500Name, boolean requireClose);
```

Comment:
I think based on the slack thread, this can be marked @Deprecated

Reply:
Marked as deprecated